www/blak05/sem-2/main.js

```diff
@@ -0,0 +1,99 @@
+$(document).ready(function(){
+
+    var baseApiLyrics = 'https://orion.apiseeds.com/api/music/lyric/'; 
+    var apiKeyLyrics = '?apikey=cRomUgw9cay6obVEueVwhiNZTGQFlqBv9evFRe1Xx1frFDCqy7leXKmiEfEMfcID';
+    var baseApiChart = 'http://ws.audioscrobbler.com/2.0/?method=chart.gettoptracks&api_key=c0fd8b3e8900c8cb6ec278f750bcf6cb&format=json';
+    loadChartAjax();
+    submit.addEventListener('submit', function(e) {
+ 
+        e.preventDefault();
+        var artist = $('#artist').val();
+        var song = $('#song').val();
+        
+        $('#lyric').empty();    
+        $('#loader').show();
+ 
+        if(!artist){
```

Comment:
@potatocz co se stane kdyz zadam jen mezery?

Reply:
Nic ti to nenajde a napíše to, že hledaná píseň nebyla nalezena.